assets/prefabs/structureTemplates/types/laboratory1.prefab

```diff
@@ -0,0 +1,3 @@
+{
+    "StructureTemplateType" : {}
```

Comment:
File should be renamed to `laboratoryLevel1` according to existing naming schemes.